### DISCUSSION TITLE
This app is not published in the Google Play console

### DISCUSSION THREAD

**Discussion by haashem** [codemagic-ci-cd/codemagic-docs#1160] [Q&A] [answered]
Hi,
I have connected code magic with google play console, but when the is publishing to google play, the build gets failed with below error and no other information!
`This app is not published in the Google Play console`

**although the app is successfully published to internal testing track.**

Build ID: `6374137355aa073d6b91b079`

<img width="1041" alt="Screenshot 2022-11-15 at 22 48 14" src="https://user-images.githubusercontent.com/3206748/202041140-0cc1c8b1-f681-4e53-a70c-6cfc976e138a.png">

**Comment by himesh-cm** [comment, marked as answer]
Hi @haashem, Thanks for sharing the build info, i will check and get back to you.

**Reply by himesh-cm** [reply]
Hi @haashem,
The first version of the app has to be published manually on the play store and then the subsequent version will be automated in your CI/CD Workflow.

**Reply by jonchui** [reply]
I got the same error. When i uploaded the `aab` file that codemagic created w/ the `flutter release .... ` build, i got this error from google play console:

`You uploaded an APK or Android App Bundle that was signed in debug mode. You need to sign your APK or Android App Bundle in release mode. Learn more about signing.`

![image](https://user-images.githubusercontent.com/204868/202876652-1baf19a4-9dae-4f1f-a089-e4aa64aa21c9.png)

Here's my codemagic command:

![image](https://user-images.githubusercontent.com/204868/202876680-710ab5f0-0685-4f03-af6f-f6219d46f8ee.png)

I'm sure I'm doing something wrong? 

In the meantime, i'm just opening in android studio and running "generate a signed build"...

**Comment by haashem** [comment]
Thanks a lot for your quick reply. 
So it should be documents in Google play error list in code magic documentation. 
Also the CI should be log correct message instead of saying it’s failed, but intact has been successful. 

Thanks
